packages/api-docs-builder/utils/createTypeScriptProject.ts

```diff
@@ -22,9 +22,14 @@ export interface CreateTypeScriptProjectOptions {
   /**
    * File used as root of the package.
    * This property is used to gather the exports of the project.
-   * The path must be relative to the root path.
+   *
+   * Use an array to target more than one entrypoint.
+   *
+   * @example 'src/index.d.ts'
+   * @example ['src/index.d.ts', 'src/PigmentStack/PigmentStack.tsx']
+   *          `PigmentStack` cannot be included in the `index.d.ts` file because it is using Pigment CSS specific API.
    */
-  entryPointPath?: string;
+  entryPointPath?: string | string[];
```

Comment:
@mui/docs-infra there is a change related to docs-infra. I appreciate another eye to make sure I don't break other project.

Reply:
I don't see a way it could break other projects.

I tried it on mui-x and the proptypes & docs:api script do not modify anything  👍

Reply:
Thanks @alexfauquette